GN_annotation_wrapper/GN_annotation_wrapper.py

```diff
@@ -0,0 +1,154 @@
+import os
+import sys
+import subprocess
+import filecmp
+import csv
+import shutil
+import argparse
+
+def split(infile,anfile,unanfile):
+	comment_lines = str()
+	ann_data = str()
+	unann_data = str()
+
+	with open(infile,'r') as file:
+		for line in file:
+			if line.startswith('#'):
+				comment_lines += line
+			elif line.upper().startswith('HUGO_SYMBOL'):
+				header = line
+				cols = line.split("\t")
+				hgvsp_index = cols.index("HGVSp_Short")
+			else:
+				data = line.split("\t")
+				if data[hgvsp_index] == "":
+					unann_data += line
+				else:
+					ann_data += line
+						
+	if unann_data != "":
+		ann_file = open(anfile,"w")
+		ann_file.write(comment_lines)
+		ann_file.write(header)
+		ann_file.write(ann_data)
+		ann_file.close()
+		unann_file = open(unanfile,"w")
+		unann_file.write(comment_lines)
+		unann_file.write(header)
+		unann_file.write(unann_data)
+		unann_file.close()
+
+def get_header(file):
+	with open(file, "r") as header_source:
+		for line in header_source:
+			if not line.startswith("#"):
+				header = line
+				break
+	return header
+		
+def merge(file1,output_file):
+	out_file = open(output_file,"a+")
+	if os.path.getsize(output_file) == 0:
+		with open(file1,'r') as file:
+			for line in file:
+				out_file.write(line)
+	else:				
+		header1 = get_header(file1)
+		header2 = get_header(output_file)
+	
+		if header1 != header2:
+			print("Can't merge the files since the MAF columns are not the same")
+			sys.exit()
+
+		else:
+			with open(file1,'r') as file:
+				for line in file:
+					if line.startswith('#') or line.upper().startswith('HUGO_SYMBOL') :
+						continue
+					else:
+						out_file.write(line)
+	out_file.close()
+
+def rearrange_mafcols(ref_file,unan_file):
+	ref_header = get_header(ref_file)
+	unan_header = get_header(unan_file)
+
+	if ref_header != unan_header:
+		file_name = unan_file[0:-4]
```

Comment:
there is a built in `os` command that gives you the basename of a file.  `os.path.basename(filename)`